src/index.html

```diff
@@ -12,7 +15,39 @@
       href="styles/index.scss"
```

Comment:
The `href` attribute in your link tag is pointing to a SCSS file. This won't work because browsers can't interpret SCSS. You should compile your SCSS to CSS and link to the compiled CSS file instead.